scripts/contentScript.js

```diff
@@ -380,6 +380,154 @@ async function FilesInPrBitbucket(response) {
 	});
 }
 
+const githubHunkHighlight = async (apiResponses) => {
+
+	// TODO: optimization  needed to not highlight next deleted line space if not present in response.
+	let getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
+	getFileName.forEach(async (item) => {
+		let fileContent = item.getAttribute('data-tagsearch-path');
+		if (fileContent) {
+
+			const matchEncrypted = await sha256(fileContent);
+			const foundFile = apiResponses.find(item => item.filepath === matchEncrypted);
+
+			if (foundFile) {
+				// checking for Diff view either undefined or split 
+				const deletedLines = document.querySelectorAll('input[value]');
+				let diffView = false;
+				deletedLines.forEach((item) => {
+					const getValue = item.getAttribute('value');
+					const getName = item.getAttribute('checked');
+
+					if (getValue == 'unified' || getValue == 'split') {
+						if (getName == 'checked' && getValue == 'unified') {
+							diffView = true; // for undefined View
+						}
+
+					}
+
+				});
+
+				const value = Array.from(item.getElementsByTagName('tr'));
+
+				if (diffView) {
+					// for undefined View 
+					let flag = false;
+					value.forEach((item, index) => {
+						const deletedLines = item.querySelector('button[data-original-line]')
+						if (deletedLines !== null) {
+							const originalLine = deletedLines.getAttribute('data-original-line');
+							const signature = originalLine.charAt(0);
+							const tableNumber = item.querySelector('td[data-line-number]');
+							const checkNumber = tableNumber.getAttribute('data-line-number');
+							if ((signature == '-' || signature == '+') && checkNumber >= foundFile.line_start && checkNumber <= foundFile.line_end) {
+								flag = true;
+							} else {
+								flag = false;
+							}
+
+							if (flag) {
+								item.style.backgroundColor = 'rgb(86, 88, 0)';
```

Comment:
Since we are using the same colour multiple times, I suggest that we make a global constant for that at the start of this file.